client/hbs/disqus.hbs

```diff
@@ -1,20 +1,25 @@
 <div id="disqus_thread" class="disqus"></div>
 <script>
 /**
-* RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
-* LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
-*/
-var disqus_config = function () {
+ * RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
+ * LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables
+ */
+var disqus_config = function() {
 	this.page.url = dttw.tech; // Replace PAGE_URL with your page's canonical URL variable
-	this.page.identifier = {{this}}; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
+	this.page.identifier = {
+		{
+			this
+		}
+	}; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
```

Comment:
Also, I'm pretty sure `{ this }` is legal as of ES6 (the result is `{ "this": <this> }`), but I'm not sure what wrapping it in another object would do.